### PR TITLE
Fix broken link in README

### PR DESCRIPTION
**Pull request by victoryforce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ sudo make install
 
 ##### Windows
 
-See https://github.com/darktable-org/darktable/blob/master/packaging/windows/BUILD.md
+See https://github.com/darktable-org/darktable/tree/master/packaging/windows
```
Comment (TurboGit) [inline README.md:335]: Probably better to only change BUILD.md to README.md. This way it displays the procedure directly.

Comment (kmilos) [reply README.md:335]: Actually, the reason it was renamed to README.md was so it was not needed, and the directory is enough - GitHub renders the text then automatically., so I also prefer this link style.

Comment (TurboGit) [reply README.md:335]: @kmilos : I see, I didn't know that the default README.md was read when opening any dir. Works for me then, thanks to both of you.

Comment (kmilos) [reply README.md:335]: Yep, on GitHub (and GitLab as well?) it is very convenient like that, any README.md/README.txt/README is rendered as the dir page...

**Pull request merged**